opendevin/runtime/server/runtime.py

```diff
@@ -114,9 +112,6 @@ async def browse(self, action: BrowseURLAction) -> Observation:
     async def browse_interactive(self, action: BrowseInteractiveAction) -> Observation:
         return await browse(action, self.browser)
 
-    async def recall(self, action: AgentRecallAction) -> Observation:
-        return NullObservation('')
```

Comment:
`recall` always return `NullObservation` which means it is not really useful...

Reply:
Only as a sidenote:
That action is supposed to call `search_memory` method from within the current `agent` file.
It is a stub in the `ServerRuntime` because in the base `Runtime` class it's an `@abstractmethod`.

However, it is indeed questionable, whether this action will be of use in the future and it currently isn't correctly dealt with in `agent_controller`, neither.

Reply:
I think we can plan to remove `search_memory` once @enyst's memory refractor is in place.. I plan to do it in this PR but i think it is actually get used in the monologue agent..